src/plugins/management_overview/public/index.ts

```diff
@@ -6,6 +6,6 @@
 import { ManagementOverViewPlugin } from './plugin';
 
 export { OverviewApp } from './overview_app';
-export { ManagementOverViewPluginSetup, ManagementOverViewPluginStart } from './plugin';
+export { ManagementOverViewPluginSetup } from './plugin';
```

Comment:
I seem to have it in my head that the convention is to always export a PluginStart method, even if it's empty. @ashwin-pc @AMoo-Miki Do you remember if it's important one way or another?

Reply:
I'm not sure of the implications of not exporting it, but it that's what we've done so far, let's keep it that way for consistency

Reply:
to make it constitency, move it back.